### PR TITLE
add digital object identifier class

### PR DESCRIPTION
**Pull request by maxheld83**

### CODE REVIEW

File: reprex/doi.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+There's a `knit.print()` method, too:
+
+-   [`https://doi.org/10.1038/nphys1170`](https://doi.org/10.1038/nphys1170)
+-   `NA`
+-   [`https://doi.org/10.1007/978-3-642-65840-2_5`](https://doi.org/10.1007/978-3-642-65840-2_5)
+
+``` r
+# much easier to enter this way
```
Comment (krlmlr) [inline reprex/doi.md:21]: Interleaving reprexes are more difficult to run locally, should we repeat the `library()` call here?

```suggestion
library(biblids)
# much easier to enter this way
```

**Review by krlmlr — APPROVED** [review]
Thanks, very nice!

File: reprex/doi.md (hunk 1 above)
Comment (maxheld83) [reply reprex/doi.md:22]: yes, thanks for catching that!

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**